plugins/webhooks/index.js

```diff
@@ -76,10 +78,19 @@ const webhooksPlugin = {
 
                         parsed.pluginOptions = pluginOptions;
 
-                        const { type, hookId } = parsed;
+                        const { type } = parsed;
+                        hookId = parsed.hookId;
 
                         request.log(['webhook', hookId], `Received event type ${type}`);
 
+                        if (queueWebhookEnabled) {
```

Comment:
We have to check not only `QUEUE_WEBHOOK_ENABLED` is true but also executor-queue executor is used.

Reply:
Since a conditional branch by the executor name should not be done at this handler, I fixed to start event without enqueuing when enqueueWebhook is not implemented.
https://github.com/screwdriver-cd/screwdriver/pull/2627/commits/60e01aef16d800d1ba71c523a9c62ba71dc05a88